common:
    lidar_topic:  "/livox/lidar"
    imu_topic:    "/livox/imu"
    image_topic:  "/camera/color/image_raw"
    image_type:   RGB8    # 1 RGB8   2 COMPRESSED
    gravity_acc: [ 0.0, 0.0, 9.81 ]

lidar_parameter:
    lidar_type: 1                # 1 for Livox serials LiDAR
    N_SCANS: 6
    SCAN_RATE: 10                # unit: Hz
    time_unit: 3                 # 0-second, 1-ms, 2-us, 3-ns

imu_parameter:
    acc_cov: 0.1
    gyr_cov: 0.1
    b_acc_cov: 0.0001
    b_gyr_cov: 0.0001
    time_diff_enable: false

camera_parameter:
    image_width: 1280
    image_height: 800
    image_resize_ratio: 0.5
    camera_intrinsic: [ 1000.0, 0.0, 640.0,
                        0.0, 1000.0, 400.0,
                        0.0, 0.0, 1.0 ]
    camera_dist_coeffs: [ 0.0, 0.0, 0.0, 0.0, 0.0 ]  #k1, k2, p1, p2, k3

extrinsic_parameter:
    extrinsic_enable: false
    extrinsic_t_imu_lidar: [ 0.0, 0.0, 0.0 ]
    extrinsic_R_imu_lidar: [ 1, 0, 0,
                             0, 1, 0,
                             0, 0, 1 ]
    extrinsic_t_imu_camera: [ 0.0, 0.0, 0.0 ]
    extrinsic_R_imu_camera: [ 1, 0, 0,
                              0, 1, 0,
                              0, 0, 1 ]
